content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -16,24 +16,108 @@ your model training using Iterative Studio.
 
 Iterative Studio uses your regular CI/CD setup (e.g. GitHub Actions) to run the
 experiments. This means that to enable experimentation from Iterative Studio,
-you need to integrate your Git repository with a CI/CD setup (e.g. GitHub
-Actions).
+you need to do the following:
 
-Then, on each Git commit, the CI/CD setup will get invoked. If this setup
-includes a model training process, it will be triggered, which means that your
-ML experiment will run automatically.
+1. First, integrate your Git repository with a CI/CD setup that includes model
+   training process. You can do this in one of two ways:
 
-For more details on how to set up
-[CI/CD pipelines for your ML project](/doc/use-cases/ci-cd-for-machine-learning),
-refer to [CML](https://cml.dev). You can use any cloud or Kubernetes for the
-model training process. CML also generates a report after the CI/CD setup
-executes.
+   - by
+     [using the wizard provided by Iterative Studio](#use-the-iterative-studio-wizard-to-set-up-your-ci-action).
+   - by creating the CI set up on your own. For this, create workflow files
+     (such as GitHub Actions) that get triggered on push or pull request. For
+     more details on how to set up
+     [CI/CD pipelines for your ML project](/doc/use-cases/ci-cd-for-machine-learning),
+     refer to [CML](https://cml.dev).
+
+2. Then,
+   [submit your experiment from Iterative Studio](#submit-a-new-experiment). The
+   submission will invoke your CI/CD setup, triggering the model training
+   process.
 
 > Due to access restrictions, you cannot run experiments on the demo project
 > (`example-get-started`) that is provided to you by default. Once you connect
 > to your ML project repositories, you can follow the instructions given below
 > to run experiments directly from Iterative Studio.
 
+## Use the Iterative Studio wizard to set up your CI action
+
+When you select one of the commits in your project table and click on the `Run`
+button, an input form will open. At the top of this form, you will see a small
+message that invites you to set up your CI in case you have not done it yet.
+
+![](https://static.iterative.ai/img/studio/set_up_cml_message.png)
+
+Click on `Click here` to use the wizard to create your CML workflow within your
+CI/CD vendor.
+
+The wizard UI is divided in two sections:
```

Comment:
Too verbose?

```suggestion
The CML wizard is divided in two sections:
```

Reply:
> Click on `Click here`

It feels like that should be "self-documenting" -- we don't need to explain to users that "click here" means they should click there.

Bottom line: Studio is a visual product so it should be intuitive, then we don't need to document every step in such detail because it just repeats the UI.

Reply:
Updated. And I'll see what else can be shortened.